smac/tae/execute_func.py

```diff
@@ -43,13 +43,15 @@ def __init__(
         par_factor: int = 1,
         cost_for_crash: float = float(MAXINT),
         abort_on_first_run_crash: bool = False,
+        n_workers: int = 1,
```

Comment:
Would it be possible that this is an argument only to the dask worker?

Reply:
Done